src/updater/main.ts

```diff
@@ -23,17 +24,13 @@ let updateData: UpdateData;
 
 handle(IpcEvents.UPDATER_GET_DATA, () => updateData);
 handle(IpcEvents.UPDATER_DOWNLOAD, () => {
-    const portable = !!process.env.PORTABLE_EXECUTABLE_FILE;
-
     const { assets } = updateData.release;
     const url = (() => {
         switch (process.platform) {
             case "win32":
                 return assets.find(a => {
-                    if (!a.name.endsWith(".exe")) return false;
-
-                    const isSetup = a.name.includes("Setup");
-                    return portable ? !isSetup : isSetup;
+                    if (PORTABLE && a.name.endsWith("win.zip")) return true;
+                    if (!PORTABLE && a.name.endsWith(".exe")) return true;
```

Comment:
this block should properly return a boolean instead of undefined
```suggestion
                    return PORTABLE ? a.name.endsWith("win.zip") : a.name.endsWith(".exe");
```

Reply:
Same logic either way, but that's definitely a clever and precise way to put it! Change made.